Cache/IO.lean

```diff
@@ -169,6 +169,7 @@ def unpackCache (hashMap : HashMap) : IO Unit := do
     IO.println s!"Decompressing {size} file(s)"
     let isMathlibRoot ← isMathlibRoot
     hashMap.forM fun path hash => do
+      let _ ← IO.asTask do
```

Comment:
We should be waiting for the tasks to finish afterwards.

Reply:
Whoops, sorry; my review was premature.

What does Lean do about zombie tasks? Does the runtime wait for them to finish, or are they interrupted?

Reply:
Currently they are always waited for

Reply:
...though it is unclear to me whether that is the correct default

Reply:
As a compromise, can the runtiem emit a warning?

Reply:
> Currently they are always waited for

Unless you call `Process.exit`.